src/components/Theme/style.css

```diff
@@ -41,6 +41,7 @@
   display: flex;
   overflow: scroll;
   -webkit-overflow-scrolling: touch;
+  -ms-overflow-style: none;
```

Comment:
how does it work on the doc selection screen in case it needs to scroll?

Reply:
We have noticed some UI issues with that but we fixed them here: https://github.com/onfido/onfido-sdk-ui/pull/588

Reply:
Content is still scrollable, but the bar is not visible @rfreitas